pkg/asset/installconfig/aws/endpoints.go

```diff
@@ -28,17 +28,72 @@ const (
 	AwsIsoBPartitionID  = "aws-iso-b"  // AWS ISOB (US) partition.
 )
 
+// AWS Standard partition's regions.
+const (
```

Comment:
I thought that this was already added. Was this in the destroy PR that was temporarily removed?

Typically values like this will go in pkg/types/aws